students/jakielczyk_miroslaw/lesson_10_organizing_files/deleting_unneeded_files.py

```diff
@@ -0,0 +1,10 @@
+import os
+
+root_folder = 'folder_1'
+print('Files that are bigger than 12KB')
+for folder_name, sub_folders, file_names in os.walk(root_folder):
+    for filename in file_names:
+        file_size = os.path.getsize(os.path.join(folder_name, filename)) // 1024
+        if file_size > 12:
```

Comment:
Would be better to use CONSTANT var on top of the script instead of hard-coded magic value `12`
the same like `root_folder`